pages.hi/common/git-init.md

```diff
@@ -1,12 +1,20 @@
 # git init
 
 > एक नया स्थानीय गिट रिपॉजिटरी शुरू करता है।
-> अधिक जानकारी: <https://git-scm.com/docs/git-init>.
+> अधिक जानकारी: <https://git-scm.com/docs/git-init>।
 
 - एक नया स्थानीय भंडार शुरू करें:
 
 `git init`
 
-- एक नंगे हड्डी के भंडार को शुरू करें, जो ssh के रिमोट के रूप में उपयोग के लिए उपयुक्त है:
+- प्रारंभिक शाखा के लिए निर्दिष्ट नाम के साथ एक भंडार शुरू करें:
+
+`git init --initial-branch={{branch_name}}`
```

Comment:
would be great if you could translate `branch_name` to Hindi as well considering other commands also follows the same scheme.This would introduce consistency across the docs.

Reply:
done